### ISSUE TITLE
Outline does not work if file does not have ini extension

### ISSUE THREAD

**Issue by selivan** [DavidWang88/ini-for-vscode#11]
If file is opened in `ini` language mode, but does not have `.ini` extension, Outline section does not work.

Environment:

```
davidwang.ini-for-vscode:

Version: v0.0.4

VSCode:

Version: 1.55.2
Commit: 3c4e3df9e89829dce27b7b5c24508306b151f30d
Date: 2021-04-13T09:37:02.931Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
```

**Comment by trgardos** [comment]
Indeed, if you set the file association to "ini" in setting, the syntax highlighting works but section folding doesn't. If you add ".ini" extension the same file, then section folding works.

**Comment by akhiljalagam** [comment]
Is it possible to enable the Outline section in ini language mode regardless of the file extension, rather than relying solely on the presence of the .ini extension? As a workaround, I currently use symlink files with the .ini extension.
